orca-queue-redis/src/main/kotlin/com/netflix/spinnaker/orca/q/redis/RedisQueue.kt

```diff
@@ -83,7 +82,17 @@ class RedisQueue(
         ?.also { id ->
           val ack = this::ackMessage.partially1(id)
           redis.readMessage(id) { message ->
-            callback(message, ack)
+            val attempts = message.getAttribute(AttemptsAttribute::class.java)!!.attempts
```

Comment:
Not sure how useful it'd be to add this to the interface but, maybe:

`inline fun <reified A : Attribute> getAttribute(clazz: Class<A>, default: A): A`?